initializers/power_feeds.yml

```diff
@@ -0,0 +1,16 @@
+# - name: power feed 1
+#   #site: AMS 1
+#   power_panel: power panel AMS 1
+#   voltage: 208
+#   amperage: 50
+#   max_utilization: 80
+#   phase: Single phase
+#   rack: rack-01
+# - name: power feed 2
+#   #site: SING 1
```

Comment:
if `site` is allowed, it'd be great to have one `site` which is not commented in order to test the scripts.

(Our test script removes `# ` from all the initializer yamls and then starts netbox to see if they all load without error.)